### PR TITLE
Adding NRLS language model project

### PR DESCRIPTION
**Pull request by chrismainey**

### CODE REVIEW

File: projects/incident-language-model.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+remote_theme: nhsx/nhsx-io-theme
+title: Exploring large-scale language models with NHS incident data
+description: NHSX PhD Internship - Incident Language Model
+permalink: /incident-language-model/
+---
+
+# Exploring large-scale language models with NHS incident data
+
+**Keywords:** Natural Language Processing, Healthcare Text, Dataset, Embeddings, Clustering, Neural Networks
```
Comment (danjscho) [inline projects/incident-language-model.md]: `Healthcare Text, Dataset` -> `Healthcare Text Dataset`?

File: projects/incident-language-model.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+**Need:** NHS England and NHS Improvement collect a national repository of incident reporting data for England and Wales.  These data are used to detect emerging patient safety incidents and drive learning that improves patient safety.  Although the as some categorical data fields, the real 'signals' in these data are within free-text field.  Clinical experienced teams review the most servious events, and emerging themes, but the scale of the dataset means that only a few percent can recieve a full review.  Natural Language Processing (NLP) techniques have the potential to unlock learning from data that are not currently reviewed.  The dataset is currently being used for topic modelling and other analyses, but a large scale language model is not yet in use.  The creation of an appropriate language model will allow the use of clustering and other methods to identify novel targets for clinical review.  This project would seek to build the first stages of a language model by applying different representations of text to find preferred models.
```
Comment (danjscho) [inline projects/incident-language-model.md]: `Although the as some categorical` -> `Although there are some categorical`
` Clinical experienced teams` -> ` Clinically experienced teams`
`currently reviewed` -> `currently fully reviewed`
`by applying different representations` -> `by constructing different representations`

File: projects/incident-language-model.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+**Current Knowledge/Examples & Possible Techniques/Approaches:** Text extraction, processing, vector representations and weighting methods.  Use of modern open source NLP tools and exposure to neural network frameworks such as TensorFlow or Pytorch.
+
+**Related Previous Internship Projects:** n/a as first year of the scheme.  Possibility of architecture already put in place.
+
+**Enables Future Work:** This may lead to ongoing projects of where this work will be the foundation e.g. tuning the selected language model, training on more data, development of production models, examining clusting or anomoly detection algorithms.
```
Comment (danjscho) [inline projects/incident-language-model.md]: `projects of where` -> `projects where`
`clusting` -> `clustering`
`anomoly` -> `anomaly`

File: projects/incident-language-model.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+**Outcome/Learning Objectives:** Open source published code for training the language model, and potentially the language model itself.  The first step is to demonstrate the process on a smaller amount of data before applying to the full dataset.
```
Comment (danjscho) [inline projects/incident-language-model.md]: `for training the language model` -> `for fine-tuning or training the language model`?

Add comment about `comparing to a simpler benchmark model`.

**Review by danjscho — CHANGES_REQUESTED** [review]
Looks good - just a few typos and minor clarification points added.

File: projects/incident-language-model.md (hunk 2 above)
Comment (chrismainey) [reply projects/incident-language-model.md]: All of those, but `are not currently reviewed` -> `do not receive a full clinical review`

File: projects/incident-language-model.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+**Keywords:** Natural Language Processing, Healthcare Text Dataset, Embeddings, Clustering, Neural Networks
+
+**Need:** NHS England and NHS Improvement collect a national repository of incident reporting data for England and Wales.  These data are used to detect emerging patient safety incidents and drive learning that improves patient safety.  Although there are some categorical data fields, the real 'signals' in these data are within free-text field.  Clinically experienced teams review the most servious events, and emerging themes, but the scale of the dataset means that only a few percent can receive a full review.  Natural Language Processing (NLP) techniques have the potential to unlock learning from data that do not receive a full clinical review.  The dataset is currently being used for topic modelling and other analyses, but a large scale language model is not yet in use.  The creation of an appropriate language model will allow the use of clustering and other methods to identify novel targets for clinical review.  This project would seek to build the first stages of a language model by constructing different representations of text to find preferred models.
```
Comment (danjscho) [inline projects/incident-language-model.md:12]: `servious` -> `serious`?

**Review by danjscho — APPROVED** [review]

**Pull request merged**